payjoin/src/send/error.rs

```diff
@@ -269,19 +269,23 @@ impl From<InternalCreateRequestError> for CreateRequestError {
     fn from(value: InternalCreateRequestError) -> Self { CreateRequestError(value) }
 }
 
-/// Represent an error returned by the receiver.
+/// Represent an error returned by Payjoin receiver.
 pub enum ResponseError {
-    /// `WellKnown` errors following the BIP78 spec
-    /// https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#user-content-Receivers_well_known_errors
-    /// These errors are displayed to end users.
+    /// Errors that are defined in the [`BIP78`] spec.
     ///
-    /// The `WellKnownError` represents `errorCode` and `message`.
+    /// It is safe to display these errors to end users.
+    ///
+    /// [`BIP78`]: https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#user-content-Receivers_well_known_errors
     WellKnown(WellKnownError),
-    /// `Unrecognized` errors are errors that are not well known and are only displayed in debug logs.
-    /// They are not displayed to end users.
+    /// Errors out of the scope of the [`BIP78`] spec.
+    ///
+    /// These errors are only displayed in debug logs.
+    ///
+    /// [`BIP78`]: https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#user-content-Receivers_well_known_errors
```

Comment:
I prefer the original first sentence. It would be good for this to mirror the first change you made

"They are not displayed to end users." -> "Unrecognized errors are not safe to display to end users because they could be used maliciously to phish a non technical user. Only display them in debug logs."

Adding the link is a great idea.